### ISSUE TITLE
Setting 'className' prop on HotKeys causes TypeScript error TS2322

### ISSUE THREAD

**Issue by TAGC** [chrisui/react-hotkeys#59]
In a *.tsx file, I get a TypeScript error trying to set the `className` prop on a `HotKeys` component.

**Code**

```js
<HotKeys className='foo' handlers={this.hotKeyHandlers}>
   ...
</HotKeys>
```

**Error**

```
Error:(71, 20) TS2322:Type '{ className: "foo"; handlers: { saveDocument: () => void; }; children: Element[]; }' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<HotKeys> & Readonly<{ children?: ReactNode; }> & R...'.
  Property 'className' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<HotKeys> & Readonly<{ children?: ReactNode; }> & R...'.
```

**Comment by TAGC** [comment]
@chrisui Any feedback on this? It's causing me a fair amount of grief as I'm not sure of a way to suppress this type error within my actual React code without causing real errors to occur.

This can be resolved for my case by adding `className?: string` to `HotKeysProps`:

```js
interface HotKeysProps extends Props<HotKeys> {
  onFocus?: Function;
  onBlur?: Function;
  keyMap?: Object;
  handlers?: Object;
  focused?: boolean;
  attach?: any;
  className?: string;  // Added this.
}
```

A more general solution might be to extend React's `HTMLProps`:

```js
interface HotKeysProps extends Props<HotKeys>, HTMLProps<HotKeys> {
  keyMap?: Object;
  handlers?: Object;
  focused?: boolean;
  attach?: any;
}
```

This also seems to work fine for my use case (just using `keyMap` and `handlers`), but I'm not sure if it will break anything.

**Comment by chrisui** [comment]
@TAGC I'm not very familiar with typescript so not sure what we'd have to do to solve this issue.

Happy to accept a pull request and ship a hotfix straight away if you have a solution!

CC @admmasters who contributed the original typings https://github.com/chrisui/react-hotkeys/pull/28

**Comment by TAGC** [comment]
@chrisui Cheers, I've opened #60.

**Comment by progrape** [comment]
@chrisui @TAGC hi, I got the same Typescript error when setting `component` prop on `HotKeys` component, because `HotKeysProps` interface in `index.d.ts` file missing `component` declaration. Can you fix this problem together?

```jsx
<HotKeys component='g' handlers={this.hotKeyHandlers}>
   ...
</HotKeys>
```

**Comment by TAGC** [comment]
@progrape I've updated the definition in my pull request.

**Comment by progrape** [comment]
@TAGC Thanks 👍

**Comment by greena13** [comment]
Fixed in #60, I believe.
